messages/messages.go

```diff
@@ -95,6 +97,24 @@ func FormatDuration(d time.Duration) string {
 	return sb.String()
 }
 
+// Deletes a specified message after a certain time
+func ExpireMessage(b *bot.Bot, s *discordgo.Session, msg *discordgo.Message, warning string) {
+	go func() {
+		time.Sleep(15 * time.Second)
+		err := s.ChannelMessageDelete(msg.ChannelID, msg.ID)
+		if err != nil {
+			log := b.Log.With(
+				"guild_id", msg.GuildID,
+				"channel_id", msg.ChannelID,
+				"message_id", msg.ID,
+				"error", err,
+			)
+
+			log.Warn(warning)
+		}
+	}()
+}
```

Comment:
1. This should be in `internal/dgoutils`
2. There's no need to have a warning string here, just make it a standard like "failed to expire a message"